modules/sync/protocolbuffers/gofeatures/state.json

```diff
@@ -19,6 +19,10 @@
     {
       "name": "v1.36.0",
       "digest": "4e137ba5a00eb6c515866947ca789565e673f30d276e3f04565c162d712497d5c7947167b50b2558a84683c6691f2ba6c1b8a354ff356a9584c9e15635122b60"
+    },
+    {
+      "name": "v1.36.1",
+      "digest": "e24c074714bbfe83a51ec4930388d0b3e24c0052f390c92d4fcd8640ac7c67b7017f6d0643e3fb3d63e9bd359040661270cf51d1404fbea7e535cdc83d187d43"
```

Comment:
The cas dir contains only 2 new files: the new manifest and the new README. All good.